Description: set /usr/share/ruby-jsduck as the root_dir
  At least two subdirectories of root_dir are being used (template-min &
  js-classes). The current value was very gem-specific and in the package it
  was set to "/usr/lib/ruby", an obviously wrong choice. Hardcode our value,
  this should probably be made configurable before pushing upstream.
Author: Faidon Liambotis <faidon@wikimedia.org>
Last-Update: 2014-04-22

--- a/lib/jsduck/options.rb
+++ b/lib/jsduck/options.rb
@@ -136,7 +136,7 @@ module JsDuck
       @warnings_exit_nonzero = false
       @cache = false
       @cache_dir = nil
-      @root_dir = File.dirname(File.dirname(File.dirname(__FILE__)))
+      @root_dir = "/usr/share/ruby-jsduck"
       @template_dir = @root_dir + "/template-min"
       @template_links = false
       @extjs_path = "extjs/ext-all.js"
